packages/vats/src/bridge.js

```diff
@@ -11,7 +11,7 @@ export const BridgeHandlerI = M.interface('BridgeHandler', {
   fromBridge: M.call(M.any()).returns(M.promise()),
 });
 
-export const BridgeScopedManagerI = M.interface('ScopedBridgeManager', {
+export const BridgeChannelI = M.interface('ScopedBridgeManager', {
```

Comment:
```suggestion
export const BridgeChannelI = M.interface('BridgeChannel', {
```

Reply:
Changing the exported name of this interface guard to `BridgeScopedManagerI` is a (breaking) API change; are you talking about walking this part back too?

Reply:
Maybe I'm aiming for something less strict than "API compatible".  Like "state compatible".